tools/protoxform/BUILD

```diff
@@ -30,7 +30,7 @@ py_binary(
     ],
     data = [
         "//:.clang-format",
-        "//:API_VERSION",
+        "//:API_VERSION.txt",
```

Comment:
Changes like this suggest to me that users of the envoy and envoy API sources may be using these files in interesting ways. One last ask, could you please add a changelog entry so these two renames are clearly highlighted? Otherwise looks great, TY.

Reply:
Done, lmk if I put it in the wrong place!